.github/workflows/ci.yml

```diff
@@ -57,8 +87,37 @@ jobs:
       - uses: dart-lang/setup-dart@v1
         with: {sdk: stable}
       - run: dart --version
+
+      - name: Read package.json
```

Comment:
Is there a way to avoid the repetition here with a YAML alias, or is there something about GitHub Actions or this specific config that's preventing that?

Reply:
https://github.community/t/support-for-yaml-anchors/16128/41 :sob: